src/test/java/team/baymax/logic/parser/patient/DeletePatientCommandParserTest.java

```diff
@@ -27,7 +27,6 @@ public void parse_validArgs_returnsDeleteCommand() {
 
     @Test
     public void parse_invalidArgs_throwsParseException() {
-        assertParseFailure(parser, "a", String.format(MESSAGE_INVALID_COMMAND_FORMAT,
-                DeletePatientCommand.MESSAGE_USAGE));
+        assertParseFailure(parser, "a", MESSAGE_INVALID_INDEX);
```

Comment:
If you pass in a `String`, it should be invalid command format? But i think it's ok here.